### PR TITLE
Images layers extension CI

### PR DESCRIPTION
**Pull request by feloy**

### CODE REVIEW

File: .github/workflows/build-next.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
Comment (benoitf) [inline .github/workflows/build-next.yaml:33]: it's 20

File: .github/workflows/pr-check.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          - os: "windows-2022"
+          - os: "ubuntu-22.04"
+    timeout-minutes: 20
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
Comment (benoitf) [inline .github/workflows/pr-check.yaml:38]: 20

File: .github/workflows/pr-check.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+jobs:
+  build:
+    name: Build / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: "windows-2022"
+          - os: "ubuntu-22.04"
```
Comment (benoitf) [inline .github/workflows/pr-check.yaml:30]: now we've switched to 24.04

File: .github/workflows/pr-check.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        if: ${{ matrix.os=='windows-2022' }}
+        id: yarn-cache-windows
+        with:
+          path: ${{ steps.yarn-cache-dir-path-windows.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - uses: actions/cache@v3
+        if: ${{ matrix.os=='ubuntu-22.04' }}
```
Comment (benoitf) [inline .github/workflows/pr-check.yaml:64]: now we've switched to 24.04

File: .github/workflows/release.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      branch:
+        description: 'Branch to use for the release'
+        required: true
+        default: main
+env:
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
```
Comment (benoitf) [inline .github/workflows/release.yaml:35]: now we've switched to 24.04

File: .github/workflows/release.yaml [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+          gh pr ready "${pullRequestUrl}"
+          echo "🔅 Mark the PR as being ok to be merged automatically"
+          gh pr merge "${pullRequestUrl}" --auto --rebase
+          sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${CURRENT_VERSION}\",#g" package.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
Comment (benoitf) [inline .github/workflows/release.yaml:102]: 20

File: .eslintrc.json [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    "browser": false
+  },
+  "extends": [
+    "eslint:recommended",
+    /** @see https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs */
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:etc/recommended",
+    "plugin:sonarjs/recommended"
```
Comment (jeffmaury) [inline .eslintrc.json:15]: We use recommanded-legacy in other repos

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**